nglview/adaptor.py

```diff
@@ -358,13 +372,14 @@ def __init__(self, structure):
 
     def get_structure_string(self):
         # only write 1st model
-        with NamedTemporaryFile(suffix='.pdb') as fh:
+        with mkstemp_wrapper(suffix='.pdb') as fname:
             if self.only_save_1st_model:
                 self._structure.write_pdb(
-                    fh.name, coordinates=self._structure.coordinates)
+                    fname, coordinates=self._structure.coordinates)
             else:
-                self._structure.write_pdb(fh.name)
-            return fh.read().decode()
+                self._structure.write_pdb(fname)
+            with open(fname) as fh:
+                return fh.read()
```

Comment:
so we don't need to `decode` here?

Reply:
When opening the file in binary mode, it should be added. One can also do the following, which is basically the same:

```python
with open(fname, "rb") as fh:
    return fh.read().decode()
```

I'm not sure if this is also correct for Python 2 (and how relevant that still is).

Reply:
Thanks. I will test your branch tonight.